addons/web/static/tests/legacy/views/list_tests.js

```diff
@@ -12807,6 +12807,28 @@ QUnit.module('Views', {
         list.destroy();
     });
 
+    QUnit.test('FieldColorPicker: dont overflow color picker in list', async function (assert) {
+        assert.expect(1);
+
+        const list = await createView({
+            View: ListView,
+            model: 'foo',
+            data: this.data,
+            arch: `
+                <tree editable="top" string="Partners">
+                    <field name="date"/>
+                    <field name="int_field" widget="color_picker"/>
+                </tree>`,
+            domain: [['id', '<', 0]],
+       });
+        const result = list.renderer._getColumnWidth(list.renderer.columns[1])
```

Comment:
We'd prefer if you don't check the result like that. We tend to do more black box integration tests and just check the actual dom result. 
So do your asserts on the width of the colum directly.